onnxruntime/python/session.py

```diff
@@ -155,11 +155,11 @@ def __init__(self, path_or_bytes, sess_options=None, providers=[]):
         """
         self._path_or_bytes = path_or_bytes
         self._sess_options = sess_options
-        self._load_model(providers)
+        self._load_model(providers or [])
```

Comment:
Is there an advantage of doing it this way vs the previous approach?
Or is stylistic?

Reply:
this is a typical python error, some explanation could be found here.
https://nikos7am.com/posts/mutable-default-arguments/#:~:text=Avoid%20using%20an%20empty%20list%20as%20a%20default%20argument%20to%20a%20function,-June%204%2C%202018&text=This%20is%20not%20the%20behavior,time%20the%20function%20is%20called.

Reply:
that example is showing the case where one mutates the default (leading to wrong results), but the previous code wasn't doing that. But your change seems like a good pattern to follow regardless.  
thanks.